### DISCUSSION TITLE
Is it possible to control manifest mime type

### DISCUSSION THREAD

**Discussion by yotamN** [containers/buildah#5781] [Q&A]
Hey,
I'm creating a multi platform manifest using `buildah-manifest-create` and get an oci image manifest according to `buildah-manifest-inspect`. Is there a way to choose to use Docker manifest instead?

**Comment by nalind** [comment]
The format can be forced by adding an image to it that uses information that can only be represented in  Docker format (for example, using the `--features` flag, though the field it sets is largely unused and ignored).  I tend to not fret about the local copy, since it can be converted, along with the images listed in it, if it's pushed it to a registry with the `--format docker` flag.
